### PR TITLE
#4144 enable to change the default version of services used by Config…

### PR DESCRIPTION
**Pull request by chia7712**
…urator image

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4144)
I have tested this PR on my local and the it succeeds to change the version
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docker/configurator.dockerfile [hunk 1]
```diff
@@ -22,11 +22,17 @@ ARG REPO="https://github.com/oharastream/ohara.git"
 ARG BEFORE_BUILD=""
 ARG KAFKA_VERSION=2.4.0
 ARG SCALA_VERSION=2.12
+# this argument is used to change default version of services
+ARG OHARA_VERSION=""
```
Comment (chia7712) [inline docker/configurator.dockerfile]: @jackyoh 麻煩把此參數加到jenkins~

Comment (jackyoh) [reply docker/configurator.dockerfile]: ok

Comment (jackyoh) [reply docker/configurator.dockerfile]: 我已經加入到 jenkins, docker-compose 的地方了。--build-arg OHARA_VERSION=$tag

**Comment by chia7712** [comment]
retry clientIT

**Comment by chia7712** [comment]
unrelated error

**Pull request merged**